src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -646,6 +639,9 @@ def finalize(self) -> None:
     def free(self) -> None:
         pass
 
+    def execute(self) -> None:
+        pass
+
```

Comment:
Why do we need this?

Reply:
This was added as an abstract method to `BaseQueryCompiler` in upstream Modin last year as a means of forcing computation for lazy backends (https://github.com/modin-project/modin/issues/5221) that all children classes have to implement. Even though our DataFrame is lazy it would take a fair bit of work to actually properly implement this, so I elected to just leave a `pass` for now.

Reply:
Gotcha, makes sense.